ajax/libs/weblas/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "weblas",
+  "description": "GPU accelerated BLAS for node and the browser",
+  "filename": "weblas.js",
```

Comment:
Let's use `weblas.min.js`
Because the minified tool still can minify the file.
Thank you.